src/test/kotlin/com/terraformation/backend/db/DatabaseBackedTest.kt

```diff
@@ -2391,22 +2387,22 @@ abstract class DatabaseBackedTest {
   protected fun insertDocument(
       createdBy: UserId = inserted.userId,
       createdTime: Instant = Instant.EPOCH,
-      methodologyId: MethodologyId = cannedMethodologyId,
+      documentTemplateId: DocumentTemplateId = inserted.documentTemplateId,
```

Comment:
Mostly to make it consistent, but also because after removing [these lines](https://github.com/terraware/terraware-server/pull/2050/files#diff-e911a186ebc1e23d7bef146f87e2215e9937047ad719f93b5184883af2e57833L102), some controller integration tests starting failing.